.pre-commit-config.yaml

```diff
@@ -14,17 +14,11 @@ repos:
   hooks:
     - id: isort
       name: isort (python)
-- repo: https://github.com/ambv/black
```

Comment:
Is there a particular reason you removed black?

Reply:
Yeah we talked about using code formatters last year and decided that between flake8 and isort we were happy with code quality.

https://github.com/django-cms/django-cms/discussions/7125